packages/opds-web-client/src/components/OPDSCatalog.tsx

```diff
@@ -18,30 +17,26 @@ export interface OPDSCatalogProps {
   epubReaderUrlTemplate?: (epubUrl: string) => string;
 }
 
-/** The main application component. */
-export default class OPDSCatalog extends React.Component<OPDSCatalogProps, {}> {
-  store: Redux.Store<State>;
-  context: NavigateContext;
+/**
+ * The main application component.
+ *  - Renders root and passes props along with store to root
+ *  - Creates the redux store using OPDSStore
+ *  - consumes and passes it down
```

Comment:
Small typo, "consumes" should be capitalized to be consistent with the rest of the list.